src/linear_algebra/multilinear.lean

```diff
@@ -0,0 +1,255 @@
+/-
+Copyright (c) 2019 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel
+-/
+
+import linear_algebra.basic
+
+/-!
+# Multilinear maps
+
+We define multilinear maps as maps from `ι → M₁` to `M₂` which are linear in each coordinate. Here,
```

Comment:
What is `iota`? An arbitrary type? A fintype?

Reply:
Any type. I have added this in the docstring.